### DISCUSSION TITLE
Version 2.980

### DISCUSSION THREAD

**Discussion by gzsombor** [SkyTubeTeam/SkyTube#1028] [General]
* Update NewPipeExtractor to fix recent Youtube changes - by Chris Bellew
* Fix a couple of crashes
* Dependency updates
* Updated translations for:
  * Catalan - by DFC
  * Croatian - by Milo Ivir
  * Finnish - by Ricky Tigg
  * French - by J. Lavoie
  * German - by J. Lavoie and Sascha Werth
  * Japan - by Takaki Iekura
  * Portugal - by ssantos
  * Spanish - by Julio Montero
  * Simplified Chinese - by Eric
  * Tamil - by Pugal Krishnan
  * Turkish - by Oğuz Ersen
  * Ukrainian - by Ihor Hordiichuk
  * Vietnamese - by bruh
 

<hr /><em>This discussion was created from the release <a href='https://github.com/SkyTubeTeam/SkyTube/releases/tag/v2.980'>Version 2.980</a>.</em>

**Comment by gaijin213** [comment]
Hi,
Thx for the hard work!
Anyway you can fix the CSV files import issue plz?

**Comment by manilenio** [comment]
I'm trying to make this work for my car's head unit with android 4.3. So far, here are my observations:
version 2.978 - plays but no video nor audio (unable to disable new type files)
version 2.979 - won't play anything (new type files disabled/enabled but no change)
version 2.980 - Featured page is blank; plays video but no audio (new type files disabled)

I hope the next version will finally be the one that will work because this seems to have better implementation than Newpipe. Anyway, fingers crossed.
